### ISSUE TITLE
Test SAMBA share before creating snapshot

### ISSUE THREAD

**Issue by ronsum232** [thomasmauerer/hassio-addons#6]
In trying to setup the add-on, I encountered difficulties connecting to a Win10 share. Every attempt to re-configure the add-on and server share required the creation of a new snapshot. I suppose I could have tested using a partial snapshot, but figure the add-on should test write capabilities with the share, and log its status.

Great work on this add-on!

**Comment by thomasmauerer** [comment]
I would rather suggest to extend the docu with points to be taken care of during configuration. What exactly was your problem? Missing write permissions?

**Comment by ronsum232** [comment]
My first attempt at using the add-on, I created a Win 10 share and configured the add-on - It failed, but only after creating a full snapshot.
A few attempts at re-configuring both Win 10 share and add-on happened before I figured out that Win 10 sharing wizard doesn't set file system security permissions for the folder shared - fixed that.

Reason for opening issue:
The add-on's default config performs a full snapshot (excludes not set).  Each troubleshooting test of the add-on will report smb access issues after a snapshot is taken... and this can become a very lengthy process.

Layperson's ideas:
1. Add-on tests ability to write to configured share at startup (or before starting snapshot)

or

2. Make doing a partial snapshot a lot easier by implementing **exclude_addons: all** and present that as the example config or maybe **include_addons:** .

I only intended to offer feedback and my appreciation. Thanks again.
